cmd/proxy/cmd/proxy.go

```diff
@@ -89,6 +89,9 @@ type ClusterChecker struct {
 	endPollonProxyCh chan error
 
 	pollonMutex sync.Mutex
+
+	checkIntervalSeconds  int
+	requestTimeoutSeconds int
```

Comment:
Just use time.Duration instead of an int. So you can avoid all the conversions done in the other parts of the code you changed.